src/Sdk/DTPipelines/Pipelines/ActionStepDefinitionReference.cs

```diff
@@ -150,4 +153,36 @@ public override ActionStepDefinitionReference Clone()
             return new ScriptReference(this);
         }
     }
+
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
copied this class from the server

Reply:
actually.... i don't think we need this anymore. Is this because originally we had a `checkout` keyword?